src/app/utils/utils.js

```diff
@@ -102,11 +102,11 @@ function Utils() {
   };
 
   /**
-   * upperCaseFirstChar(str)
+   * capitalizeFirstChar(str)
    * Makes the first character uppercase in a string
    * @param {string} str
    */
-  this.upperCaseFirstChar = (str) => {
+  this.capitalizeFirstChar = (str) => {
```

Comment:
Ah, capitalize means making the first char uppercase, so there's no need to also put `FirstChar`.

Reply:
ah hahaha OK!